src/core/api/public_api.ts

```diff
@@ -2268,6 +2270,14 @@ class Player extends EventEmitter<IPublicAPIEvent> {
    */
   private _priv_onPlaybackEvent(event : IInitEvent) : void {
     switch (event.type) {
+      case "inband-events":
+        const inbandEvents = event.value;
+        const eventNbr = inbandEvents.length;
+        for (let i = 0; i < eventNbr; i++) {
+          const inbandEvent = inbandEvents[i];
+          this.trigger("inbandEvent", inbandEvent);
```

Comment:
Isn't there a risk that the application chose to stop/load a new content as an inbandEvent arrives in which case we would send the next one although that content doesn't play anymore?